code/controllers/ProcessScheduler/README.md

```diff
@@ -1,86 +0,0 @@
-ProcessScheduler
```

Comment:
this is a well documented system, what are you replacing it with?

Reply:
> this is a well documented system, what are you replacing it with?

MC Processing.